src/lib.rs

```diff
@@ -74,21 +74,28 @@ extern crate error_chain;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde;
-#[macro_use] 
+#[macro_use]
 extern crate serde_json;
+#[macro_use]
+extern crate log;
 
 extern crate handlebars;
 extern crate pulldown_cmark;
 extern crate regex;
 
+#[cfg(test)]
 #[macro_use]
-extern crate log;
+extern crate pretty_assertions;
+#[cfg(test)]
+extern crate tempdir;
```

Comment:
I found the `pretty_assertions` crate invaluable when you try to debug assertions where the `assert!()` is comparing two complex structures. However you have a point in that it's not really worth adding yet another dependency for a feature which is only used 3 or 4 times... I think I'll remove it and see how I go without it.